src/index.js

```diff
@@ -259,7 +261,13 @@ class FsDatastore {
    * @returns {PullStream}
    */
   query (q /* : Query<Buffer> */) /* : QueryResult<Buffer> */ {
-    let tasks = [glob(path.join(this.path, '**', '*' + this.opts.extension))]
+    // glob expects a POSIX path
+    let prefix = q.prefix || '**'
+    let pattern =
+      path.join(this.path, prefix, '*' + this.opts.extension)
+      .split(path.sep)
+      .join('/')
```

Comment:
shouldn't this be the inverse? Split by the key separator `/` and the join with the os specific path sep?

Reply:
Beg to differ.  The `path.join` creates an os specific path. The `split(path.sep)` then gives an array from the os specific and then `join('/')` creates a POSIX specific path, which is what `glob` wants,

Reply:
ah `glob.sync` does expect POSIX [`/`](https://github.com/isaacs/node-glob#windows]) separators - makes sense 👍